packages/frontend/src/hooks/useSpaces.ts

```diff
@@ -66,7 +66,7 @@ export const useDeleteMutation = (projectUuid: string) => {
             onSuccess: async () => {
                 await queryClient.refetchQueries(['spaces', projectUuid]);
                 history.push({
-                    pathname: `/projects/${projectUuid}/home`,
+                    pathname: `/projects/${projectUuid}/spaces`,
```

Comment:
This mutation could be  used anywhere, so we should not assume we are deleting spaces from /spaces page . If we want to fix  #4665 we simply need to remove this redirection.